Source/CarthageKit/GitHub.swift

```diff
@@ -83,6 +83,15 @@ public struct GitHubRelease: Equatable {
 	/// The name of this release.
 	public let name: String
 
+	/// The name of this release, with fallback to its tag when the name is an emtpy string
+	public var nameWithFallback: String {
+		if name.isEmpty {
+			return tag
+		} else {
+			return name
```

Comment:
Could you use a ternary operator instead of the if statement for this please? i.e. `name.isEmpty ? name : tag`.